src/commands/folding.rs

```diff
@@ -0,0 +1,70 @@
+/*
+ *   Copyright (c) 2020 Owen Salter <owen@devosmium.xyz>
+ *   All rights reserved.
+ */
+
+use crate::prelude::*;
+extern crate reqwest;
```

Comment:
`extern crate` declarations belong in `main.rs`, and `reqwest` needs to be added as a depdency in `Cargo.toml`